modules/mono/csharp_script.cpp

```diff
@@ -120,6 +120,7 @@ void CSharpLanguage::init() {
 	GLOBAL_DEF("dotnet/project/assembly_name", "");
 #ifdef TOOLS_ENABLED
 	GLOBAL_DEF("dotnet/project/solution_directory", "");
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "dotnet/project/assembly_reload_attempts", PROPERTY_HINT_RANGE, "1,16,1,or_greater"), 3);
```

Comment:
Should this be an Editor setting instead?

```c++
EDITOR_DEF("dotnet/project/assembly_reload_attempts", 3);
EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "dotnet/project/assembly_reload_attempts", PROPERTY_HINT_RANGE, "1,16,1,or_greater"));
```

Reply:
Could be either, but I was thinking that if you have a reason to change this value, it is likely because something in the project takes a long time to unload and thus for that project only it should be increased by a bunch.

Reply:
I guess that makes sense. I wish we could override editor settings per project (https://github.com/godotengine/godot-proposals/issues/1480).